product-service/src/functions/get-products-by-id/mockproductlist.ts

```diff
@@ -0,0 +1,58 @@
+export default [
```

Comment:
I noticed that this data array is repeated in different place. You could move it to higher directory and share among your lambdas.